keyboards/projectkb/signature87/config.h

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2015 Jun Wako <wakojun@gmail.com>
```

Comment:
2015...?

Reply:
@mechmerlin is a time traveler, obviously. :D